LibreHardwareMonitorLib/Hardware/RyzenSMU.cs

```diff
@@ -0,0 +1,625 @@
+                    switch (_pm_table_version)
+                    {
+                        case 0x240902:
+                            _pm_table_size = 0x514;
+                            break;
+                        case 0x240903:
+                            _pm_table_size = 0x518;
+                            break;
+                        case 0x240802:
+                            _pm_table_size = 0x7E0;
+                            break;
+                        case 0x240803:
+                            _pm_table_size = 0x7E4;
+                            break;
+                        default:
+                            return false;
+                    }
+                    break;
+                case CpuCodeName.VERMEER:
+                    switch (_pm_table_version)
+                    {
+                        case 0x2D0903:
+                            _pm_table_size = 0x594;
+                            break;
+                        case 0x380904:
+                            _pm_table_size = 0x5A4;
+                            break;
+                        case 0x2D0803:
+                            _pm_table_size = 0x894;
+                            break;
+                        case 0x380804:
+                            _pm_table_size = 0x8A4;
+                            break;
+                        default:
+                            return false;
+                    }
+                    break;
+                case CpuCodeName.RENOIR:
+                    switch (_pm_table_version)
+                    {
+                        case 0x370000:
+                            _pm_table_size = 0x794;
+                            break;
+                        case 0x370001:
+                            _pm_table_size = 0x884;
+                            break;
+                        case 0x370002:
+                        case 0x370003:
+                            _pm_table_size = 0x88C;
+                            break;
+                        case 0x370004:
+                            _pm_table_size = 0x8AC;
+                            break;
+                        case 0x370005:
+                            _pm_table_size = 0x8C8;
+                            break;
+                        default:
+                            return false;
+                    }
+                    break;
+                case CpuCodeName.PICASSO:
+                case CpuCodeName.RAVENRIDGE:
+                case CpuCodeName.RAVENRIDGE2:
+                    _pm_table_size_alt = 0xA4;
+                    _pm_table_size = 0x608 + _pm_table_size_alt;
+                    break;
+                default:
+                    return false;
+            }
+
+            return true;
+        }
+
+        private bool GetPmTableVersion(ref uint version) 
+        {
+            uint[] args = { 0 };
+            uint fn;
+
+            switch (_cpu_code_name) {
```

Comment:
Add braces to switches to follow code style.

Reply:
Also in other switches.

Reply:
Sure, fixed